src/components/Navigation/NavBar.js

```diff
@@ -53,14 +54,31 @@ LanguageItem.propTypes = {
   language: PropTypes.oneOf(languages).isRequired
 };
 
+const setFilterAndLanguage = (props, eventKey) => {
+  props.setLanguage(eventKey);
+  const filter = {};
+  Object.keys(props.filter).forEach(group => {
+    filter[group] = {};
+    Object.keys(props.filter[group]).forEach(tag => {
+      if(group === 'language' && tag === eventKey){
+        filter[group][tag] = true;
+      }else{
+        filter[group][tag] = false;
+      }
+    });
+  });
+  props.resetFilter();
```

Comment:
I stedet for setFilterAndLanguage, kunne man ikke enten utvidet `resetFilter` litt slik at den tok inn (optionally) én groupName og én tagName (begge kan være undefined), og dersom de er satt, så er det kun den ene kombinasjonen som får true? Se resetFilter i src/reducers/filter.js. Nå gjør denne funksjonen nesten det samme som resetFilters gjør.